notebooks/introduction_to_tensorflow/labs/4_keras_functional_api.ipynb

```diff
@@ -311,7 +311,7 @@
     "\n",
     "evalds = create_dataset(\n",
     "    pattern=\"../data/taxi-valid*\", batch_size=BATCH_SIZE, mode=\"eval\"\n",
-    ").take(NUM_EVAL_EXAMPLES // 1000)"
+    ").take(NUM_EVAL_EXAMPLES // BATCH_SIZE)"
```

Comment:
Dividing by batch size makes more sense since the eval dataset uses the same batch size as training. Also, too many magic number confuses students.